### ISSUE TITLE
Fails to launch on first startup.

### ISSUE THREAD

**Issue by TinFoilFox** [comixed/comixed#447]
I've been trying to get Comixed to work for the past few hours now. I've tried on Windows 10, WSL, and Ubuntu.
On each system I get the same error when starting.
```
2020-07-27 00:56:32.142  INFO 11744 --- [           main] org.comixedproject.ComiXedApp            : Starting ComiXedApp v0.7.0-0 on Harrison-Desktop with PID 11744 (C:\Users\Harrison\Desktop\comixed-app-0.7.0-0\bin\comixed-app-0.7.0-0.jar started by Harrison in C:\Users\Harrison\Desktop\comixed-app-0.7.0-0\bin)
2020-07-27 00:56:32.152 DEBUG 11744 --- [           main] org.comixedproject.ComiXedApp            : Running with Spring Boot v2.3.1.RELEASE, Spring v5.2.7.RELEASE
2020-07-27 00:56:32.153  INFO 11744 --- [           main] org.comixedproject.ComiXedApp            : No active profile set, falling back to default profiles: default
2020-07-27 00:56:34.327  INFO 11744 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFERRED mode.
2020-07-27 00:56:35.180  INFO 11744 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 842ms. Found 15 JPA repository interfaces.
2020-07-27 00:56:35.782  INFO 11744 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.security.access.expression.method.DefaultMethodSecurityExpressionHandler@e84a8e1' of type [org.springframework.security.access.expression.method.DefaultMethodSecurityExpressionHandler] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2020-07-27 00:56:35.787  INFO 11744 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'methodSecurityMetadataSource' of type [org.springframework.security.access.method.DelegatingMethodSecurityMetadataSource] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2020-07-27 00:56:36.206  INFO 11744 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 7171 (http)
2020-07-27 00:56:36.214  INFO 11744 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-07-27 00:56:36.215  INFO 11744 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.36]
2020-07-27 00:56:36.316  INFO 11744 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-07-27 00:56:36.317  INFO 11744 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 4062 ms
2020-07-27 00:56:36.394 DEBUG 11744 --- [           main] o.c.w.a.ComiXedAuthenticationFilter      : Filter 'comiXedAuthenticationFilter' configured for use
2020-07-27 00:56:36.497 ERROR 11744 --- [           main] com.zaxxer.hikari.HikariConfig           : Failed to load driver class org.h2.Driver from HikariConfig class classloader org.springframework.boot.loader.LaunchedURLClassLoader@45283ce2
2020-07-27 00:56:36.498  WARN 11744 --- [           main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [org/springframework/boot/autoconfigure/liquibase/LiquibaseAutoConfiguration$LiquibaseConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [liquibase.integration.spring.SpringLiquibase]: Factory method 'liquibase' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under '' to com.zaxxer.hikari.HikariDataSource
2020-07-27 00:56:36.501  INFO 11744 --- [           main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2020-07-27 00:56:36.524  INFO 11744 --- [           main] ConditionEvaluationReportLoggingListener :

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2020-07-27 00:56:36.526 ERROR 11744 --- [           main] o.s.b.d.LoggingFailureAnalysisReporter   :

***************************
APPLICATION FAILED TO START
***************************

Description:

Failed to bind properties under '' to com.zaxxer.hikari.HikariDataSource:

    Property: driver-class-name
    Value: org.h2.Driver
    Origin: "driverClassName" from property source "source"
    Reason: Failed to load driver class org.h2.Driver in either of HikariConfig class loader or Thread context classloader

Action:

Update your application's configuration
```
I am running java jdk12 and jdk14 on all devices and I still can't seem to get the app to start.
I am running the latest Comixed release. Any advice would be welcome.

**Comment by mcpierce** [comment]
@HarryFry The issue was related to a recent change in project structure dropping the H2 JAR file. That should now be fixed in the latest 0.7.0 build. Please download that and see if it's fixed and, if so, you can close this ticket.

**Comment by TinFoilFox** [comment]
Yes that fixed it, thank you.
